src/cert.rs

```diff
@@ -83,10 +83,12 @@ pub struct Certificate {
 }
 
 impl Certificate {
-    pub(crate) fn new(private_key: String, certificate: String) -> Self {
-        Certificate {
-            private_key,
-            certificate,
+    /// Create Certificate from `String/&str` key and certificate
+    /// useful with manually read files from disk.
+    pub fn new(private_key: impl Into<String>, certificate: impl Into<String>) -> Self {
```

Comment:
The doc comment should mention that this is in PEM format to be analogous with for example `private_key` vs `private_key_der`.

Reply:
Done